redskyctl/internal/commands/results/results.go

```diff
@@ -17,157 +17,65 @@ limitations under the License.
-	DisplayURL bool
-	// IdleTimeout is the time between heartbeats to the "/health" endpoint required to keep the server up (defaults to 5 seconds)
-	IdleTimeout time.Duration
+	DisplayURL    bool
+	IdleTimeout   time.Duration
 }
 
 // NewCommand creates a new command for displaying the results UI
 func NewCommand(o *Options) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "results",
-		Short: "Serve a visualization of the results",
-
-		PreRun: func(cmd *cobra.Command, args []string) {
-			commander.SetStreams(&o.IOStreams, cmd)
-			o.Complete()
-		},
-		RunE: commander.WithContextE(o.results),
+		Use:        "results",
+		Short:      "View a visualization of the results",
+		Deprecated: "you can now access your results anytime using the web interface",
+
+		PreRun: commander.StreamsPreRun(&o.IOStreams),
+		RunE:   commander.WithoutArgsE(o.results),
 	}
 
-	cmd.Flags().StringVar(&o.ServerAddress, "address", "", "address to listen on")
+	// Keep the flags so we don't fail, but mark the all as hidden
+	cmd.Flags().StringVar(&o.ServerAddress, "address", "", "ignored for compatibility")
 	cmd.Flags().BoolVar(&o.DisplayURL, "url", false, "display the URL instead of opening a browser")
-	cmd.Flags().DurationVar(&o.IdleTimeout, "idle-timeout", 5*time.Second, "set the heartbeat interval (0 to ignore heartbeats)")
+	cmd.Flags().DurationVar(&o.IdleTimeout, "idle-timeout", 5*time.Second, "ignored for compatibility")
+	_ = cmd.Flags().MarkHidden("address")
+	_ = cmd.Flags().MarkHidden("url")
 	_ = cmd.Flags().MarkHidden("idle-timeout")
 
 	return cmd
 }
 
-func (o *Options) Complete() {
-	// Default to listening on an ephemeral port
-	if o.ServerAddress == "" {
-		o.ServerAddress = ":0"
-	}
-
-	// If we are just printing a URL we can't rely on the heartbeat to keep the process alive
-	if o.DisplayURL {
-		o.IdleTimeout = 0
-	}
-}
-
-func (o *Options) results(ctx context.Context) error {
-	// Create the router to match requests
-	router := http.NewServeMux()
-	if err := o.handleAPI(ctx, router, "/v1/"); err != nil {
-		return err
-	}
-	o.handleUI(router, "/ui/")
-	o.handleLiveness(router, "/health")
-
-	// Create the server
-	server := commander.NewContextServer(ctx, router,
-		commander.WithServerOptions(o.configureServer),
-		commander.ShutdownOnInterrupt(func() { _, _ = fmt.Fprintln(o.Out) }),
-		commander.ShutdownOnIdle(o.IdleTimeout, func() { _, _ = fmt.Fprintln(o.Out) }),
-		commander.HandleStart(o.openBrowser))
-
-	// Start the server, this will block until someone calls 'shutdown' from above
-	return server.ListenAndServe()
-}
-
-func (o *Options) configureServer(srv *http.Server) {
-	srv.Addr = o.ServerAddress
-	srv.ReadTimeout = 5 * time.Second
-	srv.WriteTimeout = 10 * time.Second
-	srv.IdleTimeout = 15 * time.Second
-}
-
-func (o *Options) openBrowser(loc string) error {
+func (o *Options) results() error {
 	u, err := user.Current()
 	if err != nil {
 		return err
 	}
 
+	loc := "https://app.stormforge.io/experiments"
```

Comment:
Should we/can we pull this in from the config? More of a dev issue

Reply:
I added configuration to `optimize-go`; I will pick it up here and in #232 once it is merged and released.